test/unit/src/loaders/LoadingManager.js

```diff
@@ -1,6 +1,22 @@
 /**
- * @author TristanVALCKE / https://github.com/TristanVALCKE
+ * @author Don McCurdy / https://github.com/donmccurdy
  */
 
-//Todo
-console.warn("Todo: Unit tests of LoadingManager")
+QUnit.module( "LoadingManager" );
+
+QUnit.test( "setResourceTransform", function( assert ) {
+
+  var manager = new THREE.LoadingManager();
+  var suffix = '?transformed=true';
+
+  manager.setResourceTransform( function ( url ) {
+
+    return url + suffix;
+
+  } );
+
+  var url = 'https://foo.bar/baz';
+  var resolvedURL = manager.resolveResourceURL( url );
+  assert.equal( resolvedURL, url + suffix, 'URL transform is applied' );
+
+});
```

Comment:
Wrong indents. (Indents should be not spaces but tabs).

Reply:
Fixed!